internal/install/recipe_installer.go

```diff
@@ -350,8 +350,7 @@ func (i *RecipeInstaller) installRecipes(ctx context.Context, m *types.Discovery
 			lastError = err
 
 			log.Debugf("Failed while executing and validating with progress for recipe name %s, detail:%s", r.Name, err)
```

Comment:
@sanderblue another "Failed" message here needing to be updated.

Reply:
Do we care about that for debug messages vs default user-facing messages?

Reply:
Ho sorry I didnt see this was debug and not the default output for users, that's fine then.